lib/erb_lint/cli.rb

```diff
@@ -33,8 +33,9 @@ def initialize
     end
 
     def run(args = ARGV)
-      load_options(args)
-      @files = args.dup
+      dupped_args = args.dup
+      load_options(dupped_args)
+      @files = dupped_args.dup
```

Comment:
Since you've already `dup`-ed on line 36, this `dup` isn't necessary (because nobody else knows about `dupped_args` and we don't use it anywhere else, so we can pass ownership over to `@files`)

Reply:
Updated!